neonvm/hack/generate.sh

```diff
@@ -21,17 +18,17 @@ git config --global --add safe.directory "$GOPATH/src/github.com/neondatabase/au
 
 echo "Running gen_helpers ..."
 kube::codegen::gen_helpers \
-    --output-base "/go/src" \
-    --input-pkg-root  github.com/neondatabase/autoscaling/neonvm/apis \
-    --boilerplate neonvm/hack/boilerplate.go.txt
+    --boilerplate neonvm/hack/boilerplate.go.txt \
+    github.com/neondatabase/autoscaling/neonvm/apis
+
 
 echo "Running gen_client ..."
 kube::codegen::gen_client \
-    --output-base "/go/src" \
-    --input-pkg-root  github.com/neondatabase/autoscaling/neonvm/apis \
-    --output-pkg-root github.com/neondatabase/autoscaling/neonvm/client \
```

Comment:
There's a comment above about generation requiring that we put things in `$GOPATH/src` -- from this, it seems like at least outdated, and _maybe_ no longer true? can you validate that?

Reply:
Definitely seems to not be true anymore. Don't know enough about code gen to say for sure, but it still works after the change so I'd say that's good enough to say it's not true.